tests/base.py

```diff
@@ -179,6 +179,14 @@ def get_admin_location_id(self):
         location = Location.query.filter_by(name=user.location).first()
         return location.id
 
+    def assign_resource(self, query):
```

Comment:
@kwanj-k ,I suppose we don't need this method since there is a method that works the same in `CommonTestCases` class.